### ISSUE TITLE
Comment folding

### ISSUE THREAD

**Issue by notduncansmith** [bennyguitar/News-YC---iPhone#40]
It would be really cool to have "Fold" as an option on the comment menu to collapse sub threads.

**Comment by bennyguitar** [comment]
Yeah, I've been meaning to add comment folding in for a looooong time now, but haven't had time to think of a great performant solution and implement it. I'll keep this issue open, as I'd definitely love to have fold/unfold options to make it easier to breeze past stuff you've already read or don't want to see.
